internal/storage/gcs/folder.go

```diff
@@ -23,15 +23,15 @@ import (
 
 type Folder struct {
 	Name           string
-	Metageneration int64
+	MetaGeneration int64
 	UpdateTime     time.Time
 }
 
 func GCSFolder(bucketName string, attrs *controlpb.Folder) *Folder {
 	// Setting the parameters in Folder and doing conversions as necessary.
 	return &Folder{
 		Name:           getFolderName(bucketName, attrs.Name),
-		Metageneration: attrs.Metageneration,
+		MetaGeneration: attrs.Metageneration,
 		UpdateTime:     attrs.GetUpdateTime().AsTime(),
 	}
```

Comment:
Why are we changing `Metageneration` to `MetaGeneration` if control folder is returning in `Metageneration` format then shouldn't we keep the same?

Reply:
no Metageneration is not correct convention, if there is incorrect naming convention in library method, we should not leak it to our code as well, also MinObject also has MetaGeneration to maintain consistency.